lib/bundler/resolver.rb

```diff
@@ -344,7 +344,7 @@ def verify_gemfile_dependencies_are_found!(requirements)
           versions = @source_requirements[name][name].map(&:version)
           message  = "Could not find gem '#{requirement}' in #{requirement.source}.\n"
           if versions.any?
-            message << "Source contains '#{name}' at: #{versions.join(", ")}"
+            message << "Source contains '#{name}' at: #{versions.join(", ")} #{requirement.__platform}"
```

Comment:
For the ruby case, we don't want to have it print `Could not find gem 'foojava (= 1.1) ruby'` or something? Isn't it possible for the target gemspec to indicate java or some other platform, but the Gemfile.lock to indicate ruby?

Ie. we'd want
```
Could not find gem 'foojava (>= 0) java' in source at ..
Source contains 'foojava' at: 1.2.1 ruby
```
instead of:
```
Could not find gem 'foojava (>= 0) java' in source at ..
Source contains 'foojava' at: 1.2.1
```

Thoughts?